lib/pennyworth/vm.rb

```diff
@@ -33,8 +37,29 @@ def stop
       @runner.stop
     end
 
-    def run_command(*args)
-      command_runner.run(*args)
+    def run(*args)
```

Comment:
In the context of the class the method name `run` is ambiguous, as it could also mean running the VM. So it might be better to keep the `run_command`.